custom.d.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+declare module '*.svg' {
```

Comment:
I can see that we have added an icon to our repository, I can see an issue that in the future if we have a lot of these our repository size will increase substantially. Isn't this icon in @twin/itwinui package with other icons or maybe there is a higher version of the package we are currently using to get this icon from the installed package?

Reply:
I did not see this icon in itwinui icons library, or some others that would match file type extensions - but all available icons are in https://ux.bentley.com/icons/

It would be better if it could come from a library. I set it up this way so it is at least simple to add new svg icons as served static files